app/models/TransportEquipment.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+case class TransportEquipment(containerId: Option[String]) {
+
+  def asString: Option[String] = containerId match {
+    case Some(containerId) => Some(s"Container $containerId")
```

Comment:
Best practice would be to define a message key for this. Can we redefine this as `def asString(implicit messages: Messages): Option[String] = ???`